docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -1,175 +1,268 @@
-cluster with the templates contained in the snapshot, replacing any existing
-ones whose name matches one in the snapshot. It completely removes all
-persistent settings, non-legacy index templates, ingest pipelines and
-{ilm-init} lifecycle policies that exist in your cluster and replaces them with
-the corresponding items from the snapshot.
-
-The restore operation must be performed on a functioning cluster. However, an
-existing index can only be restored if it's <<indices-close,closed>> and
-has the same number of shards as the index in the snapshot. The restore
-operation automatically opens restored indices if they were closed and creates
-new indices if they didn't exist in the cluster.
-
-If a data stream is restored, its backing indices are also restored. The restore
-operation automatically opens restored backing indices if they were closed.
-
-In addition to entire data streams, you can restore only specific backing
-indices from a snapshot. However, restored backing indices are not automatically
-added to any existing data streams. For example, if only the
-`.ds-logs-2099.03.08-000003` backing index is restored from a snapshot, it is
-not automatically added to the existing `logs` data stream.
+----
+// TEST[setup:setup-snapshots]
+// TEST[s/_restore/_restore?wait_for_completion=true/]
 
 [discrete]
-=== Partial restore
+[[close-restore]]
+==== Close and restore
+
+You can restore an existing index if it's closed. However, the index in the
+snapshot must have the same number of primary shards. You can't close a data
+stream.
+
+// Only display warning in 7.x.
+// In 7.x, `action.destructive_requires_name` defaults to false.
+ifeval::["{major-version}" == "7.x"]
+WARNING: Don't use the <<indices-close,close index API>> to target the `*` or
+`.*` wildcard expressions. If you use {es}'s security features, this will close
+system indices required for authentication. To close all regular indices, use
+`*,-.*`.
+endif::[]
 
-By default, the entire restore operation will fail if one or more indices or backing indices participating in the operation don't have
-snapshots of all shards available. It can occur if some shards failed to snapshot for example. It is still possible to
-restore such indices by setting `partial` to `true`. Please note, that only successfully snapshotted shards will be
-restored in this case and all missing shards will be recreated empty.
+[source,console]
+----
+# Close an index
+POST my-index/_close
 
+# Restore the index from a snapshot
+POST _snapshot/my_repository/my_snapshot_2099.05.06/_restore 
+{
+  "indices": "my-index"
+}
+----
+// TEST[setup:setup-snapshots]
+// TEST[s/_restore/_restore?wait_for_completion=true/]
 
 [discrete]
-[[change-index-settings-during-restore]]
-=== Changing index settings during restore
+[[rename-on-restore]]
+==== Rename on restore
+
+You can rename indices and data streams you restore. Before you start, ensure
```

Comment:
This is frequently used when you want to side-by-side compare old vs current index data (e.g. unexpected record updates/deletes).

Reply:
I struggled with the wording, but I added a statement to cover this.